src/main/java/zpi/controller/attraction/AttractionController.java

```diff
@@ -33,12 +34,14 @@ public ResponseEntity<Page<AttractionDto>> getAllAttractions(Pageable pageable)
         return new ResponseEntity<>(attractions, HttpStatus.OK);
     }
 
+    @PreAuthorize("isAuthenticated()")
```

Comment:
Te adnotacje to pewnie tak tylko na testa pewnie. Musimy pamiętać, żeby zrobić to legitnie później

Reply:
Tak to na testy, teraz poprawilem i dodalem tam wymog roli gdzie moim zdaniem jest potrzebny